### ISSUE TITLE
how to deploy face_recognition api on android system?

### ISSUE THREAD

**Issue by sainttelant** [ageitgey/face_recognition#229]
i followed the setup instruction installed api on Ubuntu16.04 successfully, i am also curious about how to deploy entire api on the android  just like pic of obama recognized on the cellphone i guess showed by instruction, could someone plz  demonstrate briefly how to do that ? i will be appreciate so much

**Comment by ageitgey** [comment]
This library doesn't run on Android.

**Comment by sainttelant** [comment]
@ageitgey  i see,  thanks anyway, i thought it was executed on android showed by adam, actually, i guess he just showed the pic on the cellphone to the webcam,  haha

**Comment by sainttelant** [comment]
@ageitgey , by the way,  do you intend to figure out a way to run API on cellphone in the future? i think that will be fun

**Comment by sainttelant** [comment]
@ageitgey  Dude, can you explain why can this API recognize face so precisely just by feeding one single pic to train? exciting, amazing, as we all know,  it must be needed numerous of pics to train  order to acquire higher accuracy,just like tensorflow objection_detection, i know you maybe called tensorflow during training, however, how can u do that just mere one single pic to train, and obtain corresponding pic encoding？

**Comment by sainttelant** [comment]
did you call opencv face recognition XML  or some haar feature during training as i assumed?

**Comment by AlainPilon** [comment]
There is a whole article about how dlib works in linked in the readme. 

TLDR: It is not a neural network, it takes face measurements and calculate distance of points between encodings.

**Comment by ageitgey** [comment]
Like @AlainPilon said, check out this article for how it works: https://medium.com/@ageitgey/machine-learning-is-fun-part-4-modern-face-recognition-with-deep-learning-c3cffc121d78

**Comment by sainttelant** [comment]
@ageitgey @AlainPilon , thanks fellows
